content/millennium/r4/general-clinical/procedure.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Procedure | R4 API
+---
+
+# Procedure
+
+* TOC
+{:toc}
+
+## Overview
+
+The Procedure resource returns medical and surgical procedures performed on or for a patient during their lifetime.   Historical procedures, as well as procedures recorded during a specific visit, are returned. Surgical procedures from finalized surgical cases will be returned as free text procedures if the solution has been configured to write procedures from finalized cases to Procedure History.
```

Comment:
nit: procedures, if the solution

Reply:
Updated [here](https://github.com/cerner/fhir.cerner.com/pull/322/commits/783dce5f317c0ee97b87117e79fb62dcbca23583#diff-b5510004918e501427ece368dff4f71aR12).

Reply:
+1